### ISSUE TITLE
Get a deployment on Kubernetes working

### ISSUE THREAD

**Issue by malwaredllc** [malwaredllc/minicache#13]
I want this project to be easily deployed on Kubernetes

**Comment by adesokanayo** [comment]
Can i work on this ticket ?

**Comment by malwaredllc** [comment]
@adesokanayo Absolutely. We should discuss this on Slack as the existing architecture using node-*.json config files might not work with k8s, we may need to change it

**Comment by zeina1i** [comment]
Hi guys,
Are you working on this issue at the moment? And If you don't have decision to work on it can I take it?

**Comment by malwaredllc** [comment]
Hey @zeina1i you can work on it, let. me know if you have any questions
